setup.py

```diff
@@ -105,6 +105,17 @@
 import glob
 import importlib
 
+
+# Use copies instead of symbolic files.
+# Windows has very poor support for them.
+sym_files = ['tools/shared/cwrap_common.py']
+orig_files = ['aten/src/ATen/common_with_cwrap.py']
+for sym_file, orig_file in zip(sym_files, orig_files):
+    if os.path.exists(sym_file):
+        os.remove(sym_file)
+    shutil.copyfile(orig_file, sym_file)
```

Comment:
We probably don't want to do this every time `setup.py` is ran. It should be part of the command that needs it (`build_deps`?)